mp/src/game/client/momentum/ui/PaintPreviewPanel.cpp

```diff
@@ -0,0 +1,63 @@
+#include "cbase.h"
+
+#include "PaintPreviewPanel.h"
+#include "clientmode.h"
+
+#include "mom_shareddefs.h"
+#include "util/mom_util.h"
+
+#include <vgui/ISurface.h>
+
+#include "tier0/memdbgon.h"
+
+static void PaintScaleCallback(IConVar *var, const char *pOldValue, float flOldValue);
+
+static ConVar mom_paint_color("mom_paint_color", "0000FFFF", FCVAR_CLIENTCMD_CAN_EXECUTE | FCVAR_ARCHIVE,
+                              "Amount of red on the painting textures");
```

Comment:
It's not just red, it's the actual color for the paint texture.

Reply:
what the, I guess it has always been like this